SPECS/kernel/config

```diff
@@ -7129,9 +7130,9 @@ CONFIG_TMPFS_XATTR=y
 # CONFIG_TMPFS_INODE64 is not set
 # CONFIG_TMPFS_QUOTA is not set
 CONFIG_HUGETLBFS=y
+# CONFIG_HUGETLB_PAGE_OPTIMIZE_VMEMMAP_DEFAULT_ON is not set
```

Comment:
Moved as a result of https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/5cfaddaa4bdbbf795085eff1fc52f6820e116bcb